index.md

```diff
@@ -136,14 +136,6 @@ the following JSON-LD in expanded form.
 ]
 ```
 
-## Outstanding issues
-
-Nested GeoJSON coordinate arrays are incompatible with the processing model
-of JSON-LD 1.0. There are no current best practices for addressing this issue.
-Applications that process GeoJSON as JSON-LD using the GeoJSON-LD 1.0 context
-must take extra steps not defined here to ensure that geometric coordinates
-are processed appropriately.
```

Comment:
@mejackreed @azaroth42 what would you think about, instead of removing this entire paragraph, leaving the first sentence (still true for JSON-LD 1.0) and adding another sentence explaining that it is no longer a problem with JSON-LD 1.1?

Reply:
@mejackreed @azaroth42 ping?

Reply:
That seems ok to me! Would you like me to make that change?

Reply:
Yes please 🙏

Reply:
Nevermind.